src/middlewared/middlewared/plugins/kubernetes_linux/pods.py

```diff
@@ -25,3 +28,52 @@ async def query(self, filters, options):
                 pod['events'] = events[pod['metadata']['uid']]
 
         return filter_list(pods, filters, options)
+
+    async def get_logs(self, pod, container, namespace):
+        async with api_client() as (api, context):
+            return await context['core_api'].read_namespaced_pod_log(
```

Comment:
Don't we have any sort of streaming API here? I don't like the entire log being loaded into RAM